advocacy_docs/supported-open-source/postgresql/uninstalling.mdx

```diff
@@ -51,15 +52,13 @@ These instructions apply if you used EDB's interactive Mac installer to install
 
 To uninstall PostgreSQL on a Mac system:
 
-1.  Assume the identity of an operating system superuser, and navigate into the folder where the uninstaller resides:
-
-  `/Library/PostgreSQL/<version>`
-
-  Where `version` is the version of PostgreSQL you installed.
-
 1.  Invoke the uninstaller:
 
-  `open uninstall-postgresql.app`
+    ```
+    sudo /Library/PostgreSQL/<version>/uninstall-postgresql.app/Contents/MacOS/installbuilder.sh
+    ```
+
+    Where `<version>` is the version of PostgreSQL you installed.
 
 1.  If prompted, provide a password that allows the uninstaller to make changes to your system. 
```

Comment:
1. You will be prompted for a password. If your user has admin rights on macOS, enter your password. Otherwise, enter the user name and password of a user with admin rights.

Reply:
Expanded the wording to explain what is being prompted for.